app/src/main/java/io/github/droidkaigi/confsched/fragment/SessionsTabFragment.java

```diff
@@ -162,6 +208,12 @@ public void unLiked(LikeButton likeButton) {
                     session.checked = false;
                     dao.updateChecked(session);
                     AlarmUtil.handleSessionAlarm(getContext(), session);
+
+                    if (mode == MINE) {
+                        sessions.remove(session);
+                        adapter.reset(sessions);
```

Comment:
Suggestion!
I guess it is better not to remove immediately. Bacause user might do miss tap. 
But I want to remove conflict tag immediately. What do you think?